cmd/snap-seccomp/main.go

```diff
@@ -789,28 +795,132 @@ func complainAction() seccomp.ScmpAction {
 	return seccomp.ActAllow
 }
 
+var osCreateTemp = os.CreateTemp
+
+func exportBPF(fout *os.File, filter *seccomp.ScmpFilter) (bpfLen int64, err error) {
+	errPrefixFmt := "cannot export bpf filter: %w"
```

Comment:
I recently wrote something equivalent:
```go
type errorPrefix struct {
	Prefix string
	Err    error
}

func (e errorPrefix) Error() string {
	return e.Prefix + e.Err.Error()
}

func (e errorPrefix) Unwrap() error {
	return e.Err
}

func prefixError(prefix string, e error) error {
	return errorPrefix{Prefix: prefix, Err: e}
}
```

Reply:
This was not for snapd. It's fine to leave it out if you want to keep it simple.

Reply:
Then I'll leave it out with a TODO. As I said I think it will be cool to unify how we handle error prefixes in the different part of snapd